### PR TITLE
EIP-4626 Clarify Preview Function Behavior

### PR DESCRIPTION
**Pull request by Joeysantoro**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-4626.md [hunk 1]
```diff
@@ -114,16 +114,17 @@ MUST _NOT_ revert.
...
-MUST return the *exact* amount of Vault shares that would be minted if the caller were to deposit
-a given *exact* amount of underlying assets using the `deposit` method.
 MUST be inclusive of deposit fees. Integrators should be aware of the existence of deposit fees.
-MUST _NOT_ revert.
 
-Note that any discrepancy between `shares * assetsPerShare` and `assets` SHOULD be considered slippage
-in share price or some other type of condition, meaning the depositor will lose assets by depositing.
+MUST NOT revert.
+
+Note that any unfavorable discrepancy between `convertToShares` and `previewDeposit` SHOULD be considered slippage in share price or some other type of condition, meaning the depositor will lose assets by depositing.
```
Comment (fubuloubu) [inline EIPS/eip-4626.md:127]: Note that `convertToAssets` doesn't exist yet

**Review by fubuloubu — APPROVED** [review]

File: EIPS/eip-4626.md [hunk 2]
```diff
@@ -193,16 +192,18 @@ MUST _NOT_ revert.
 
 #### previewMint
 
-Allows an on-chain or off-chain user to simulate the effects of their mint at the current block,
-given current on-chain conditions.
+Allows an on-chain or off-chain user to simulate the effects of their mint at the current block, given current on-chain conditions.
+
+MUST return as close to and no more than the exact amount of assets that would be deposited in a `mint` call in the same transaction. I.e. `mint` should return the same or fewer `assets` as `previewMint` if called in the same transaction.
```
Comment (alcueca) [inline EIPS/eip-4626.md:197]:
```suggestion
MUST return as close to and no fewer than the exact amount of assets that would be deposited in a `mint` call in the same transaction. I.e. `mint` should return the same or fewer `assets` as `previewMint` if called in the same transaction.
```

**Review by alcueca — CHANGES_REQUESTED** [review]
More is less

File: EIPS/eip-4626.md (hunk 1 above)
Comment (Joeysantoro) [reply EIPS/eip-4626.md:127]: It does now!

**Pull request merged**